github/ci/prow-deploy/files/jobs/kubevirt/kubevirtci/kubevirtci-presubmits.yaml

```diff
@@ -42,14 +42,12 @@ presubmits:
         - "/bin/bash"
         - "-c"
         - >
-            apt update &&
-            apt install gettext-base -y &&
-            wget https://dl.google.com/go/go1.13.8.linux-amd64.tar.gz -nv -S &&
-            tar -xzf go1.13.8.linux-amd64.tar.gz -C /usr/local/ &&
-            export PATH=/usr/local/go/bin:$PATH &&
-            export KUBEVIRT_PROVIDER=kind-k8s-sriov-1.17.0 &&
-            export KUBEVIRT_NUM_NODES=2;
             trap "echo teardown && make cluster-down" EXIT ERR SIGINT SIGTERM;
+            wget https://dl.google.com/go/go1.13.8.linux-amd64.tar.gz -nv -S;
+            tar -xzf go1.13.8.linux-amd64.tar.gz -C /usr/local/;
```

Comment:
Don't replace && job should fail if one of the commands does not success

Reply:
From what I saw when I removed the && if one of the commands will fail it will mask the error and wont trigger the trap
this is how I tested it:
```
**with &&**
docker run --rm quay.io/kubevirtci/golang:v20210316-d295087 /bin/bash -c "
 trap 'echo hit-trap' EXIT ERR SIGINT SIGTERM;
 echo start;
 echo df /asd && echo should-fail;
 echo end;
"
**output:**
start
df /asd
should-fail
end
hit-trap
+ EXIT_VALUE=0
+ set +o xtrace

**w/o &&**
docker run --rm quay.io/kubevirtci/golang:v20210316-d295087 /bin/bash -c "
  trap "echo hit-trap" EXIT ERR SIGINT SIGTERM;
   echo start;
   echo df /asd;
   echo should-fail;
   echo end;
"
**output:** 
start
df /asd
should-fail
end
hit-trap
+ EXIT_VALUE=0
+ set +o xtrace
```

Reply:
The golang image has golang installed. If you need specificallyl `1.13.8`, you can set in the `env` section `GIMME_GO_VERSION` to `1.14.8`.